skillbridge/src/component/BenefitSection/BenefitSection.tsx

```diff
@@ -1,10 +1,57 @@
 import React from 'react'
 import { Benefitlist } from '../Data/BenefitData'
 import { benefitIcon } from '../../assets/resource/iconResource';
-// import '../BenefitSection/BenefitSection.css'
-
+// import '../BenefitSection/BenefitSection.scss'
+import useToggleState from '../CommonFunctionalities/Custom Hook/useToggleState';
 
 function BenefitSection() {
+    
+    const [viewAll, setViewAll] = useToggleState(false);
+
+    const RenderBenefitSection=() => {
+        return(
+            <div className='d-flex flex-column gap-2'>
+            <div className="row gap-1">
+                {Benefitlist.filter(benefit => {
+                    return (benefit.id <= 3);
+                }).map((benefit) => (
```

Comment:
I didn't understand the logic behind this, why are we filtering the ids less than 3 and using map again